assets/model_monitoring/components/src/feature_importance_metrics/compute_feature_importance.py

```diff
@@ -254,15 +324,23 @@ def run(args):
             return
         log_time_and_message("Reading data in spark and converting to pandas")
         baseline_df = read_mltable_in_spark(args.baseline_data).toPandas()
+        # Drop correlation id from baseline_data if present
+        if constants.MDC_CORRELATION_ID_COLUMN in baseline_df.columns:
```

Comment:
It's possible that the user is passing in a user-defined column with the same name..! Can we add a todo to switch from removing a hardcode column to adding detection on terrible columns that only have unique values?